.circleci/config.yml

```diff
@@ -133,7 +112,12 @@ commands:
             echo 'export CARGO='$(rustup which cargo --toolchain "$RUST_NIGHTLY") >> $BASH_ENV
             # Pin the version of RUSTC used for all invocations of cargo
             echo 'export RUSTUP_TOOLCHAIN="$(cat rust-toolchain)"' >> $BASH_ENV
-
+  move_shell_setup:
+    description: Shell Setup
+    steps:
+      - run:
+          name: Setup Env
```

Comment:
nit - there is a step in "shell_setup" using the same name.  Consider using a different name for this step, for example "Move Prover setup".